e2e/utils/command-utils.ts

```diff
@@ -72,19 +72,19 @@ export function runCommand(
   try {
     const r = execSync(command, {
       cwd: tmpProjPath(),
-      stdio: ['pipe', 'pipe', 'pipe'],
+      stdio: isVerbose() ? 'inherit' : ['pipe', 'pipe', 'pipe'],
       env: {
         ...getStrippedEnvironmentVariables(),
         ...childProcessOptions?.env,
         FORCE_COLOR: 'false',
-      },
+      } as any,
       encoding: 'utf-8',
       ...childProcessOptions,
-    }).toString();
```

Comment:
@meeroslav it was actually `toString()` before, I just moved it. I'll put it back. But if `encoding` makes sure what's returned it's already a string, maybe we should change this throughout the utils.